src/Graphics/src/Graphics/Platforms/iOS/UIImageExtensions.cs

```diff
@@ -27,37 +29,75 @@ public static UIImage ScaleImage(this UIImage target, float maxWidth, float maxH
 
 		public static UIImage ScaleImage(this UIImage target, CGSize size, bool disposeOriginal = false)
 		{
-			UIGraphics.BeginImageContext(size);
-			target.Draw(new CGRect(CGPoint.Empty, size));
-			var image = UIGraphics.GetImageFromCurrentImageContext();
-			UIGraphics.EndImageContext();
-
-			if (disposeOriginal)
+			if (!OperatingSystem.IsIOSVersionAtLeast(17))
```

Comment:
Just use the new way for both chunks here.

Reply:
I have modified it